tests/integration.rs

```diff
@@ -391,20 +393,29 @@ fn getxattr(#[case] harness: Harness) {
 	let data = file.get_xattr("user.test").unwrap().unwrap();
 	let expected = b"testvalue";
 	assert_eq!(data, expected);
-
-	freebsd! {
-		// Can't use c"test" syntax, because the apply macro doesn't like it
-		let name = cstr!(b"test");
-		let num = unsafe {
-			libc::extattr_get_fd(file.as_raw_fd(), EXTATTR_NAMESPACE_USER, name.as_ptr(), std::ptr::null_mut(), 0)
-		};
-		assert_eq!(num, expected.len() as isize);
-	}
 }
 
-const EXTATTR_NAMESPACE_USER: i32 = 1;
-#[allow(unused)]
-const EXTATTR_NAMESPACE_SYSTEM: i32 = 2;
+#[cfg(target_os = "freebsd")]
+#[apply(all_images)]
+fn getxattr_freebsd(#[case] harness: Harness) {
```

Comment:
Likewise, you should change the name here.
```suggestion
fn getxattr_size(#[case] harness: Harness) {
```